Scuti/Scripts/UI/Store/Offer Details/OfferShowcasePresenter.cs

```diff
@@ -118,37 +118,47 @@ public void ResetSizeImage()
             var downloader = ImageDownloader.New(false);
             var downloads = new List<Task<Texture2D>>();
 
-             
+            int amountThumbs = 0;
 
             Data.URLs.ForEach(x => downloads.Add(downloader.Download(x)));
             thumbnailParent.gameObject.SetActive(false);
             // Downloads the iamges together and process as they finish         
-
-            try
+            while (downloads.Count > 0)
             {
-                while (downloads.Count > 0)
+                try
                 {
+                    if (amountThumbs >= 10) break;
```

Comment:
@Jpablogs8 this does not prevent them from downloading

Reply:
Remove this code as we do want to cleanup all items in download. Instead, prevent more than 10 items from being downloaded at a time.

Reply:
ok! I have an idea how to do it!